src/compounds/RequestList/RequestList.jsx

```diff
@@ -8,11 +8,18 @@ const RequestList = ({ requests }) => (
   <>
     <Title title='My Requests' size='medium' />
     <div className='rounded overflow-hidden'>
-      {requests.map((req, index) => (
+      {requests.length === 0 ? (
+        <p className='mt-2'>
+          You do not have any requests yet.
+          <br />
+          <a href='/browse'>Browse our available services</a> to create a request,
+          or start a new general request by clicking the <b>"Initiate a Request"</b> button above.
+        </p>
```

Comment:
I'm thinking this would be better served from the webstore....but for now we can leave it here.